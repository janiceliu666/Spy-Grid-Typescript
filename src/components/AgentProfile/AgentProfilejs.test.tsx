// import React from 'react';
// import { shallow } from 'enzyme';
// import AgentProfile from './AgentProfile';

// // Mock agent data
//   const mockAgent = {
//     picture: {
//       large: "https://example.com/image.jpg",
//       medium: "https://example.com/image_medium.jpg",
//       thumbnail: "https://example.com/image_thumbnail.jpg",
//     },
//     login: { username: "Agent007" },
//     name: { first: "James", last: "Bond" },
//     location: {
//       street: { number: 123, name: "Secret Street" },
//       city: "London",
//       state: "England",
//       country: "United Kingdom",
//       postcode: "007",
//       coordinates: { latitude: "50.0000", longitude: "0.0000" },
//       timezone: { offset: "+1:00", description: "Greenwich Mean Time" },
//     },
//     dob: {
//       date: "1930-11-11T00:00:00.000Z",
//       age: 93,
//     },
//     gender: "male",
//     eyeColor: "blue",
//     timezone: { offset: "James", description: "Bond" },
//   };

// describe('<AgentProfile />', () => {
//   it('matches snapshot', () => {
//     const wrapper = shallow(<AgentProfile agent={mockAgent} />);
//     expect(wrapper).toMatchSnapshot();
//   });
// });
