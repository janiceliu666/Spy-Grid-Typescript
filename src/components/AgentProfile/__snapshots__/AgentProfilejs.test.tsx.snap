// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AgentProfile /> matches snapshot 1`] = `
<div>
  <img
    alt="Agent"
    src="https://example.com/image.jpg"
  />
  <h2>
    Agent007
  </h2>
  <p>
    James
     
    Bond
  </p>
</div>
`;
